### PR TITLE
Refine DescribeIndex with a timestamp allocated by Milvus

### PR DESCRIPTION
**Pull request by xiaocai2333**
issue: #25522

### CODE REVIEW

**Comment by xiaocai2333** [comment]
rerun ut

**Comment by xiaocai2333** [comment]
rerun ut

File: internal/proxy/impl.go [hunk 1]
```diff
@@ -5651,3 +5651,27 @@ func (node *Proxy) ListClientInfos(ctx context.Context, req *proxypb.ListClientI
 		ClientInfos: clients,
 	}, nil
 }
+
+func (node *Proxy) AllocTimestamp(ctx context.Context, req *milvuspb.AllocTimestampRequest) (*milvuspb.AllocTimestampResponse, error) {
+	if !node.checkHealthy() {
+		return &milvuspb.AllocTimestampResponse{Status: unhealthyStatus()}, nil
+	}
+
+	log.Info("AllocTimestamp request receive")
```
Comment (czs007) [inline internal/proxy/impl.go:5660]: better to use rpcReceived
you can fix this in next pr

**Pull request merged**